lectures/01.intro.md

```diff
@@ -34,7 +34,7 @@
             Viktor Ketipov
         </td>
         <td>
-            <a target="_blank" href="mailto:info@kipiinteractive.com"><i class="fa fa-envelope-o"></i></a>
+            <a target="_blank" href="mailto:viktor@kipiinteractive.com"><i class="fa fa-envelope-o"></i></a>

```

Comment:
This needs to be changed in the first slides of all lectures

Reply:
I will do a separate PR with that.

Reply:
let me help you with that, focus on the UE4 intro slides